am-lib/src/main/java/uk/gov/hmcts/reform/amlib/DummyService.java

```diff
@@ -1,8 +1,33 @@
 package uk.gov.hmcts.reform.amlib;
 
+import org.jdbi.v3.core.Jdbi;
+
+import java.util.List;
+
 public class DummyService {
+    private Jdbi jdbi;
+    private String msg = "Hello Dummy Service 2";
+
+    public DummyService(String dbUrl, String user, String password) {
+        if (dbUrl == null || user == null || password == null) {
+            msg = "db url or user or pass is null";
+        } else {
+            jdbi = Jdbi.create(dbUrl, user, password);
+        }
+    }
 
     public String getHello() {
-        return "Hello Dummy Service 2";
+        StringBuffer newMsg = new StringBuffer(msg);
+
+        if (jdbi != null) {
+            List<Integer> ids = jdbi.withHandle(handle ->
```

Comment:
just as an idea, to show that something is really happening and remove the need to manually insert data, maybe add this:
```
        jdbi.useHandle(handle -> {
           handle.execute("INSERT INTO ACCESS_CONTROL (null)");
        });
```

Reply:
the code that is already there also shows that it works without inserting. 
Hym I would like to keep it as it is tbh as we dont have access to db on aat